_rules/aria-required-owned-element-bc4a75.md

```diff
@@ -0,0 +1,274 @@
+
+```html
+<div role="table">
+	<div>
+		<div>
+			<div role="row">
+				<div role="cell">my cell</div>
+				<div role="cell">and another cell</div>
+			</div>
+			<div role="row">
+				<div role="cell">my cell</div>
+				<div role="cell">and another cell</div>
+			</div>
+		</div>
+	</div>
+</div>
+```
+
+#### Passed example 6
+
+Element with role `grid` owns multiple instances of required owned elements.
+
+```html
+<div role="grid">
+	<div role="rowgroup">
+		<div role="row">
+			<div role="gridcell">1</div>
+			<div role="gridcell">Item 1</div>
+			<div role="gridcell">2 Nos.</div>
+		</div>
+	</div>
+	<div role="rowgroup">
+		<div role="row">
+			<div role="gridcell">1</div>
+			<div role="gridcell">Item 1</div>
+			<div role="gridcell">2 Nos.</div>
+		</div>
+	</div>
+</div>
+```
+
+#### Passed example 7
+
+Element with role `list` owns an instance of required owned element which has an implicit role of `listitem`.
+
+```html
+<div role="list">
+	<li>Item 1</li>
+</div>
+```
+
+#### Passed example 8
+
+Element with role `menubar` owns multiple instances of required owned elements. Role `menubar` can own elements with role `menuitem`, `menuitemcheckbox` and `menuitmeradio`.
+
+```html
+<div role="menubar">
+	<div role="menuitem">Item 1</div>
+	<div role="menuitemcheckbox">Item 2</div>
+</div>
+```
+
+#### Passed example 9
+
+Element with role `list` contains a required `listitem`, the relationship between both elements is made possible through the `aria-owns` properties.
+
+**Note:** This test case follows the definition of [owned by](#owned-by). If implemented differently, this definition could cause differences in outcome of this test case.
+
+```html
+<div role="list" aria-owns="id1"></div>
+<div id="id1" role="listitem">Item 1</div>
+```
+
+#### Passed example 10
+
+Element with role `menu` has attribute `aria-busy` set to `true`.
+
+```html
+<ul role="menu" aria-busy="true">Loading</ul>
+```
+
+#### Passed example 11
+
+Element `ul` with implicit role of `list` owns at least one instance of required owned element, in this case a `span` with `role` of `listitem`.
+
+```html
+<ul>
+	<span role="listitem">Item 1</span>
+	<li>Item 2</li> <!-- implicit role -->
+</ul>
+```
+
+### Failed
+
+#### Failed example 1
+
+Element with role `list` is missing required owned element `listiem`.
+
+```html
+<ul role="list">
+	<span>Item 1</span>
+</ul>
+```
+
+#### Failed example 2
+
+Element with role `tablist` is missing required owned element `tab`.
+
+```html
+<ol role="tablist">
+	<li role="listitem">Item 1</li>
+</ol>
+```
+
+#### Failed example 3
+
+Element with role `grid` owns required owned element `rowgroup`, which is missing its required owned element `row`.
+
+```html
+<div role="grid">
+	<div>
+		<div role="rowgroup">
+			<span>Item 1</span>
+		</div>
+		<div></div>
+	</div>
+</div>
+```
+
+#### Failed example 4
+
+The element with the semantic role of `list` owns the element with the role `tab` through the `aria-owns` property.
```

Comment:
To make it clearer why the example fails, you can add something like 
", but does not own the required elements with semantic roles `group` or `listitem`" 
to the end of the sentence